src/ray/id.h

```diff
@@ -51,7 +51,7 @@ std::ostream &operator<<(std::ostream &os, const UniqueID &id);
                                                                                         \
    private:                                                                             \
     type(const std::string &binary) { std::memcpy(id_, binary.data(), kUniqueIDSize); } \
-  };
+  } /*Requires a ; at the end where the macro is used.*/
```

Comment:
This way seems more weird. I slightly prefer to just remove the semicolons.

Reply:
Yes. Since the Macro is only used in `id_def.h` and `ray_config_def.h`, it won't break the traditions in most c++ files.